Resources/Prototypes/Voice/speech_emote_sounds.yml

```diff
@@ -36,6 +36,22 @@
     params:
       variation: 0.125
 
+- type: emoteSounds
+  id: UnisexFelinid
+  params:
+    variation: 0.125
+  sounds:
+    Scream:
+      collection: FelinidScreams
+
+- type: emoteSounds
+  id: UnisexMoth
+  sound:
+    path: /Audio/Voice/Moth/scream_moth.ogg
```

Comment:
This is the fallback sound. It should use a format similar to `sounds: Scream:` as you did for Felinids.